src/main/java/finding/brudders/model/AddressBook.java

```diff
@@ -26,13 +26,13 @@ public class AddressBook implements ReadOnlyAddressBook {
      *
      * Note that non-static init blocks are not recommended to use. There are other ways to avoid duplication
      *   among constructors.
-     */
-    {
+    */ {
```

Comment:
Should the '{' and the end of the comment be on the same line? I genuinely am not sure if this is intentional so do proceed if it is